src/trans/cpu/internal/tpm_fftw.F90

```diff
@@ -45,6 +46,7 @@ MODULE TPM_FFTW
   TYPE(FFTW_PLAN),POINTER :: FFTW_PLANS(:) => NULL()
   INTEGER(KIND=JPIM) :: N_MAX=0         ! maximum number of latitudes
   INTEGER(KIND=JPIM) :: N_MAX_PLANS=4   ! maximum number of plans for each active latitudes
+  LOGICAL            :: LALL_FFTW=.FALSE. ! T=do kfields ffts in one batch, F=do kfields ffts one at a time
```

Comment:
This new logical variable `LALL_FFTW` is configurable via a new optional argument to `SETUP_TRANS`: `LD_ALL_FFTW`
This variable is then passed to `EXEC_FFTW(..., LD_ALL)`
The routine `EXEC_FFTW` itself however apparently does not make use of this variable, i.e. an unused argument.

So I propose to revert all changes related to this variable in `TPM_FFTW`, and `SETUP_TRANS` as it brings nothing.
Changing the API to external facing routines like `SETUP_TRANS` is something we should not do lightly.
It would involve also adding tests to exercise the new API, and one also needs to consider the GPU codepath, where FFTW is not used.

Reply:
Apologies, I see now that LD_ALL is used in EXEC_FFTW after all.